mia/commands/definition.py

```diff
@@ -391,12 +391,15 @@ def download_apps():
         if not os.path.isdir(download_path):
             os.makedirs(download_path, mode=0o755)
         apk_path = os.path.join(download_path, apk_info['package_name'])
-        path, http_message = urlretrieve(apk_info['package_url'], apk_path)
+        cache_path = os.path.join(handler.get_workspace_path(), 'resources', 'apps')
```

Comment:
We can keep this cache_path for now, but I would change it to also include the CPU architecture in #72